uiowa/uiowa
# Update to metatag patch and metatag_extended_perms module

Resolves #2339. 

Also addresses a few unrelated permissions updates for splits that didn't get them.

# To test
* `composer install`
* Review file changes: https://github.com/uiowa/uiowa/pull/2341/files. For each permission in the role config files, the update should be one-to-one with the new permission having an instance of the word `tag` removed from it. Additionally, there are some other updates to those role files to correct some omissions from previous updates.
* Test that patch applies: `blt ds --site=default`
* Log in to the site and masquerade as a webmaster and make sure that you can still access the metatag settings.
